10x/README.md

```diff
@@ -24,7 +24,7 @@ cd skylab/10x/wdls/
 ## Inputs
 - `read_paths`: path to a fastq folder containing all sequencing reads for the experiment
 - `sample_def`: json file containing run information (see sample_def.json for example): 
-  - `sample_names`: name for experiment
+  - `sample_names`: name for experiment note that this **must** be the prefix for each fastq file associated with the experiment. The suffix **must** be the normal illumina fastq file suffix which describes the lane and read information.
```

Comment:
Maybe reference the file pattern here? https://github.com/10XGenomics/cellranger/blob/master/tenkit/lib/python/tenkit/fasta.py#L218